app/models/product.rb

```diff
@@ -0,0 +1,5 @@
+class Product < ApplicationRecord
+    validates :name, :description, :price, presence: true
```

Comment:
refactor this to `validates_presence_of :name, :description, :price`

Reply:
on it